src/index.ts

```diff
@@ -918,6 +918,7 @@ const deviceInfoModule: DeviceInfoModule = {
   getType,
   getTypeSync,
   getUniqueId,
+  getUniqueIdSync: getUniqueId,
```

Comment:
this is unexpected, I would expect both method signatures to exist, and not be mapped to each other?